webpack/scenes/Subscriptions/Subscriptions.scss

```diff
@@ -12,3 +12,10 @@
     margin-bottom: 0;
   }
 }
+
+.option-tooltip-container {
+  display: inline-block;
+  margin-left: 10px;
+  margin-top: 3px;
+  width: 10px;
+}
```

Comment:
Should this be in the `OptionTooltip.scss` file?

Reply:
I thought about that and decided against it because this pertained to where the tooltip is positioned on the subscriptions page and not necessary where you might want it positioned on anywhere else it might be used within the application.   I tried to keep the scss for the tooltip a little more generic so it wouldn't be hard to migrate it to PF in the future.